packages/ember-data/lib/system/model/errors.js

```diff
@@ -297,6 +284,7 @@ export default Ember.Object.extend(Ember.Enumerable, Ember.Evented, {
 
     var content = get(this, 'content').rejectBy('attribute', attribute);
     get(this, 'content').setObjects(content);
+    get(this, 'errorsByAttributeName').remove(attribute);
```

Comment:
So this would have been caught by Travis we would need to run the tests using a debug build of Ember, right?